### ISSUE TITLE
Bluetooth mouse connection loss problem

### ISSUE THREAD

**Issue by Yoshinao0511** [nurupo/vlc-pause-click-plugin#110]
## Checklist

- [x] I have went over the [Troubleshooting section](https://github.com/nurupo/vlc-pause-click-plugin#troubleshooting) and the issue still persists.

## Environment

- OS:  windows11             
- VLC version:  3.021 Vetinari   
- VLC bitness:  64 bit    
- Plugin version:  for vlc 3.0 64bit  latest version  
- Plugin bitness:  64 bit 

### Additional info

If you add this plugin to VLC media player and play VLC for a long time or leave VLC running after playback ends, other Bluetooth devices (such as Bluetooth speakers) will remain connected, but only the Bluetooth mouse will lose connection, and you will not be able to control your PC.
If you turn Bluetooth off from Settings and then turn it on again, the Bluetooth mouse connection will be restored.
I tried reinstalling the Bluetooth adapter driver and the mouse driver to find the cause, but when I changed VLC so that it would not load this plugin, the symptom was improved.
Please note that I am not good at English, so I am posting this through a translation software.
Please let me know if there are any reports on this phenomenon or plans to improve it.
Thank you.

**Comment by nurupo** [comment]
I don't think the plugin has anything to do with your Bluetooth mouse losing connection. The plugin doesn't interact with the  Bluetooth stack in any way. The plugin just receives mouse events from VLC. The VLC window always receives mouse events, no matter if this plugin is installed or not. If the plugin is installed and enabled - VLC will simply passes those mouse events to the plugin. This shouldn't be any different than if the plugin was not installed.

**Comment by Yoshinao0511** [comment]
Thank you for your reply.

After that, I continued to do various tests myself, and I found one possibility that may have caused the problem, so I would like to report it.

When you download this plugin, the file is distributed in zip format, but when you unzip this file and extract it to a dll file, there is an item called "Security" in the file's general properties tag, and it says "This file was obtained from another computer. To protect this computer, access to this file may be blocked." Next to it is a checkbox that says "Allow."

If you check this checkbox and apply it, the file will be treated the same as a general local file, and the above message and checkbox will no longer be displayed.

After changing the dll file to this state and copying it back to the VLC plugin directory, the bluetooth disconnection symptoms have not occurred so far.

It may not be a fundamental solution, but I think that the symptoms may be caused by some adverse effect on the OS's mouse monitoring event.

The problem has not occurred for more than a day, so I will continue to observe the situation for a while.

Many thanks.
